data/autoyast_sle15/autoyast_media_up_gnome.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
```

Comment:
Please, be aware, that once we upgrade os-autoinst on osd we can use variables expansion, like we do here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4176/files#diff-954e34a9a9a0683ba4ebf7de4129b6b9R7 Hence you may consider switching to registration instead of all packages DVD.

Reply:
I think both SCC and all-packages DVD should be tested for autoupgrade.

Reply:
BTW, upgrade with scc is not implemented yet at the time. So I only test autoupgrade with all-packages media here.